projects/plugins/jetpack/extensions/blocks/blog-stats/edit.js

```diff
@@ -44,6 +55,12 @@ function BlogStatsEdit( { attributes, className, setAttributes } ) {
 		);
 	}
 
+	/* Translators: Number of visitors */
+	const visitorsPlaceholder = _n( 'visitor', 'visitors', parseInt( stats ), 'jetpack', 0 );
+
+	/* Translators: Number of views */
+	const viewsPlaceholder = _n( 'hit', 'hits', parseInt( stats ), 'jetpack', 0 );
```

Comment:
The error
```
[plugins/jetpack 2:10.354] [build-production-extensions] ERROR in editor-beta.js: Translator comments have gone missing for "hit"
```
is actually coming from here. There's a detailed explanation of what's going on at https://www.npmjs.com/package/@automattic/i18n-check-webpack-plugin#lost-comments-due-to-expression-movement, but the TL;DR is that this code is getting transformed to look something like this instead:
```js
	/* Translators: Number of visitors */
	const visitorsPlaceholder = _n( 'visitor', 'visitors', parseInt( stats ), 'jetpack', 0 ),
		viewsPlaceholder = _n( 'hit', 'hits', parseInt( stats ), 'jetpack', 0 );
	/* Translators: Number of views */
```
(i.e. it deletes the first `const` and moves the second assignment into the first one, but leaves the comment behind where it was)

This seems to work
```suggestion
	const
		/* Translators: Number of visitors */
		visitorsPlaceholder = _n( 'visitor', 'visitors', parseInt( stats ), 'jetpack', 0 ),
		/* Translators: Number of views */
		viewsPlaceholder = _n( 'hit', 'hits', parseInt( stats ), 'jetpack', 0 );
```
Or this
```suggestion
	const visitorsPlaceholder =
		/* Translators: Number of visitors */
		_n( 'visitor', 'visitors', parseInt( stats ), 'jetpack', 0 );

	const viewsPlaceholder =
		/* Translators: Number of views */
		_n( 'hit', 'hits', parseInt( stats ), 'jetpack', 0 );
```

I don't know why 924d9bea didn't work. Possibly I should remove the corresponding suggestion from the doc I linked if it doesn't work anymore.

Reply:
I've moved the comment and added a parameter - hopefully that works! Thanks for taking a look. :)

Reply:
```
[plugins/jetpack 2:10.961] [build-production-extensions] ERROR in editor-beta.js: Translator comments have gone missing for "visitor"
[plugins/jetpack 2:10.961] [build-production-extensions]  - Translators: Number of visitors
[plugins/jetpack 2:10.961] [build-production-extensions] 
[plugins/jetpack 2:10.961] [build-production-extensions] ERROR in editor-beta.js: Translator comments have gone missing for "hit"
[plugins/jetpack 2:10.961] [build-production-extensions]  - Translators: Number of views
[plugins/jetpack 2:10.961] [build-production-extensions] 
```

🤔

Reply:
Unfortunately f8a461d is basically a re-do of 924d9bea which still doesn't work (see #35452 for what I know about that). Either of the suggestions I made above worked for me locally.

Reply:
Sorry, it looks like I committed the wrong thing. 🤦 It works fine locally for me too - re-added your suggestion now!